R/LSgameofchance.R

```diff
@@ -169,27 +169,27 @@ LSgameofchance   <- function(jaspResults, dataset, options, state = NULL){
 
   
   ## Summary Table
-  summaryTable <- createJaspTable(title = "Summary Table")
+  summaryTable <- createJaspTable(title = gettext("Summary Table"))
   
   summaryTable$dependOn(c("n", "k", "t", "p", "s", "check"))
   summaryTable$addCitation("JASP Team (2018). JASP (Version 0.9.2) [Computer software].")
   
-  summaryTable$addColumnInfo(name = "players",   title = "Players",   type = "string")
-  summaryTable$addColumnInfo(name = "pPoint",   title = "Pr(win a point)",   type = "string")
-  summaryTable$addColumnInfo(name = "pA",   title = "Analytical",   type = "string", 
-                             overtitle = "Pr(win the game)")
-  summaryTable$addColumnInfo(name = "pS",   title = "Simulated",   type = "string", 
-                             overtitle = "Pr(win the game)")
+  summaryTable$addColumnInfo(name = "players",  title = gettext("Players"),          type = "string")
+  summaryTable$addColumnInfo(name = "pPoint",   title = gettext("Pr(win a point)"),  type = "number")
+  summaryTable$addColumnInfo(name = "pA",       title = gettext("Analytical"),       type = "number", 
+                             overtitle = gettext("Pr(win the game)"))
+  summaryTable$addColumnInfo(name = "pS",       title = gettext("Simulated"),         type = "number", 
+                             overtitle = gettext("Pr(win the game)"))
```

Comment:
I changed the column type from `"string"` to `"number"`, so JASP can nicely format the output:
before:
![image](https://user-images.githubusercontent.com/38475991/89726524-b1ca0380-da1b-11ea-9fde-a72fffd73c66.png)
after:
![image](https://user-images.githubusercontent.com/38475991/89726530-be4e5c00-da1b-11ea-98cf-a615c585cd7e.png)